common/src/main/java/com/wynntils/features/trademarket/TradeMarketBulkSellFeature.java

```diff
@@ -56,25 +56,28 @@ public void onSellDialogueUpdated(ContainerSetSlotEvent.Pre e) {
         if (!(McUtils.mc().screen instanceof ContainerScreen containerScreen)) return;
 
         StyledText title = StyledText.fromComponent(containerScreen.getTitle());
-        if (!title.equalsString(SELL_DIALOGUE_TITLE, PartStyle.StyleType.NONE)) return;
+        // The sell title changes based on the state of the screen but always contains this string
+        if (!title.contains(SELL_DIALOGUE_TITLE)) return;
```

Comment:
Maybe register a container in ContainerModel that will match both, that way when the price match feature is fixed it won't need to duplicate this title check